win32ss/user/ntuser/desktop.c

```diff
@@ -260,6 +261,16 @@ InitDesktopImpl(VOID)
                       SynchronizationEvent,
                       FALSE);
 
+    /* Check if user has disabled window snapping */
+    DWORD dwValue = 0;
+    BOOL ret = RegReadUserSetting(L"Control Panel\\Desktop",
+                                            L"WindowArrangementActive",
+                                            REG_SZ, &dwValue, sizeof(dwValue));
+    if(ret) {
```

Comment:
Please follow the style of the surrounding code.

Reply:
Could you give me an example? I'm not sure what to change, should I drop the hungarian notation and use something like `Ret` and `EntryValue`?

Reply:
```suggestion
    if (ret)
    {
```
See https://reactos.org/wiki/Coding_Style